app/src/androidTest/java/com/github/sdp/apibootcamp/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.github.sdp.apibootcamp;
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
```

Comment:
I don't really understand what this test is for? If it is just an example test we should remove it so that the CI passes.

Reply:
This was aa leftover of example tests